testdata/p4tc_samples_outputs/meter.template

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -x
+
+set -e
+
+TC="tc"
+$TC p4template create pipeline/meter numtables 1
+
+$TC p4template create action/meter/ingress/meter_exec actid 1
+$TC p4template update action/meter/ingress/meter_exec state active
+
+$TC p4template create action/meter/ingress/drop actid 2
+$TC p4template update action/meter/ingress/drop state active
+
+$TC p4template create extern/root/Meter extid 0x1B000000 numinstances 1 tc_acl 0x1136 has_exec_method
+
+$TC p4template create extern_inst/meter/Meter/ingress.global_meter instid 1 \
+tc_numel 2048 \
+constructor param type ptype bit32 1 \
+control_path tc_key index ptype bit32 id 1 param cir ptype bit64 id 2 param cbs ptype bit64 id 3 param pir ptype bit64 id 4 param pbs ptype bit64 id 5
```

Comment:
In case of meter type being `bytes`, `cir` and `pir` should be of type `rate`
```suggestion
control_path tc_key index ptype bit32 id 1 param cir ptype rate id 2 param cbs ptype bit64 id 3 param pir ptype rate id 4 param pbs ptype bit64 id 5
```

That's because we'd like the user to specify rates in more convenient notations in the control path, for example:

```
tc p4ctrl update meter/extern/Meter/ingress.global_meter tc_key index 3 \
        param cir 500kbit param cbs 64000 param pir 1Mbit param pbs 64000
```

For a `packets` meter, the notation wouldn't make much sense, so `bit64` is correct
We put this in extern the doc, but I should've emphasised it more

Reply:
The compiler will emit bit<64> only in case of 'bytes' as well because that is what we mention in pna.p4

Reply:
Sorry, I think I didn't explain it correctly
In the case of a Meter of type `bytes`, `cir` and `pir` should be of type `rate`
In case of type `packets` everything should be `bit64`